examples/kda_example.py

```diff
@@ -0,0 +1,25 @@
+# # Meta-analysis: an example of use of the KDA method.
```

Comment:
duplicate of mkda_example.py?

Reply:
Yes, that's a duplicate of `mkda_example.py`. I thought it would be great to show one example for each technique. Would you prefer to merge both examples into one or get rid of one of them (let's say KDA)?